tracer/test/snapshots/RabbitMQTests.3_x.SchemaV1.verified.txt

```diff
@@ -1321,132 +1456,525 @@
       component: RabbitMQ,
       env: integration_tests,
       message.size: 17,
+      peer.service: rabbitmq,
+      server.address: rabbitmq,
       span.kind: producer,
-      version: 1.0.0
+      version: 1.0.0,
+      _dd.peer.service.source: server.address
     }
   },
   {
     TraceId: Id_69,
     SpanId: Id_87,
-    Name: amqp.command,
+    Name: amqp.send,
     Resource: basic.publish <default> -> hello,
     Service: Samples.RabbitMQ,
     Type: queue,
     ParentId: Id_70,
     Tags: {
-      amqp.command: basic.publish,
+      amqp.command: basic.publish,
```

Comment:
Maybe breaking, maybe fixing, not really sure about that 😅 Have you seen the `test coverage` section in the PR description? Is there something specific you think shouldn't have changed? `IsActiveScopeRabbitMQ` was always returning true as long as the parent was a rabbit span, because operation name was always the same.